testgrid/conformance/upload_e2e.py

```diff
@@ -207,11 +225,11 @@ def main(cli_args):
 
     # upload metadata, log, junit to testgrid
     print 'Uploading entry to: %s' % gcs_dir
-    upload_string(gcs_dir+'/started.json', started_json)
-    upload_string(gcs_dir+'/finished.json', finished_json)
-    upload_file(gcs_dir+'/build-log.txt', log)
-    upload_file(gcs_dir+'/artifacts/'+path.basename(junit), junit)
-
+    upload_string(gcs_dir+'/started.json', started_json, dry)
+    upload_string(gcs_dir+'/finished.json', finished_json, dry)
+    upload_file(gcs_dir+'/build-log.txt', log, dry)
+    for junit_file in junits:
+        upload_file(gcs_dir+'/artifacts/'+path.basename(junit_file), junit_file, dry)
```

Comment:
So per this logic we will upload 'n' junit files to artifacts/ in GCS bucket. I am assuming TG supports handling multiple junits?